crawler_mcp/crawlers/optimized/utils/monitoring.py

```
@@ -0,0 +1,785 @@
+        if self._vectorstore_stats:
+            report["vector_store"] = dict(self._vectorstore_stats)
+
+        return report
+
+    # Embeddings
+    def record_embeddings_stats(self, **stats: Any) -> None:
+        """Attach TEI embeddings summary to the performance report."""
+        try:
+            self._embeddings_stats.update(stats)
+        except Exception:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)